src/store/modules/app/state.ts

```diff
@@ -0,0 +1,7 @@
+const state = {
+  title: 'Agriboard',
+  drawerShown: false,
+  drawerLocked: false,
```

Comment:
I don't think so? Maybe the drawer locked? We are probably going to have to set this on the first page of the app anyways tbh, because sometimes it would be a login page and sometimes it would be the home page (where the drawer is unlocked). I think more often they would be going to a page where it is unlocked, in addition to the fact the toolbar wont be shown on the login page